client_src/js/lib/modules/printerSettings.js

```diff
@@ -220,6 +220,11 @@ class PrinterSettings {
       currentPrinter.printerName;
     document.getElementById("psPrinterURL").placeholder =
       currentPrinter.printerURL;
+    // Convert websocket url into a url object
+    const webSocketURL = new URL(currentPrinter.webSocketURL);
+    // Grab out the protocol and select it on the select box.
+    document.getElementById("psWebSocketProtocol").value =
+      webSocketURL.protocol + "//";
```

Comment:
Added this into the client so we can grab the current protocol and select it on the select dropdown.